python/cudf/cudf/core/column/string.py

```diff
@@ -767,22 +765,31 @@ def contains(
             raise NotImplementedError(
                 "unsupported value for `flags` parameter"
             )
-
-        if pat is None:
-            result_col = column.column_empty(
-                len(self._column), dtype="bool", masked=True
+        if regex and not case:
+            raise NotImplementedError(
+                "`case=False` only supported when `regex=False`"
             )
-        elif is_scalar(pat):
+
+        if is_scalar(pat):
             if regex:
                 result_col = libstrings.contains_re(self._column, pat, flags)
             else:
-                result_col = libstrings.contains(
-                    self._column, cudf.Scalar(pat, "str")
-                )
+                if case is False:
+                    input_column = libstrings.to_lower(self._column)
```

Comment:
Nit: Guessing libstrings doesn't have a `casefold`-like method right? https://docs.python.org/3/library/stdtypes.html#str.casefold

Reply:
@davidwendt is (or was this ever) on our radar?

Reply:
No.